GameData/RealismOverhaul/Engine_Configs/Astris_Config.cfg

```diff
@@ -51,7 +51,7 @@
 		}
 		CONFIG
 		{
-			// source: http://www.b14643.de/Spacerockets/Diverse/European_Rocket_engines/engines.htm
```

Comment:
Actually better, here is the new link:
http://www.b14643.de/Spacerockets/Specials/European_Rocket_engines/engines.htm

Guess bnumbers rearranged the site a bit